### ISSUE TITLE
Can't install_github(repo = "TsuPeiChiu/DNAshapeR")

### ISSUE THREAD

**Issue by hangyas** [oracle/fastr#165]
Installing the `TsuPeiChiu/DNAshapeR` package from github produces the following error:
```
An internal error occurred: "should not reach here"
```

reproduce:
```
R -e 'install.packages("devtools")'
R --R.PrintErrorStacktracesToFile=true -e 'library(devtools); install_github(repo = "TsuPeiChiu/DNAshapeR")'
```

log: https://gist.github.com/hangyas/5890f6561e491d3b318a020a1ae7f8b9

tested with: `GraalVM CE 20.3.0-dev-20200912_0211`

sessionInfo:
```
> sessionInfo()
FastR version 3.6.1 (2019-07-05)
Platform: x86_64-unknown-linux-gnu (64-bit)
Running under: Ubuntu 20.04 LTS

Matrix products: default
BLAS:   /root/graalvm-ce-java11-20.3.0-dev/languages/R/lib/libRblas.so
LAPACK: /root/graalvm-ce-java11-20.3.0-dev/languages/R/lib/libRlapack.so

locale:
[1] UTF-8/UTF-8/UTF-8/C/UTF-8/UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base
```

**Comment by steve-s** [comment]
Hello hangyas,

the issue is that FastR does not implement any other serialization format than XDR [0]. This is among many other things in our backlog and it is not clear when and if this will be implemented. It seems to be an issue in `devtools::install_github`, so you may be still able to install the package manually via `R CMD INSTALL ...` (you have to install the dependencies manually too). `devtools` is indeed quite a complex package with a large list of (also complex) dependencies. Such packages often don't work on FastR at the moment.

**Comment by hangyas** [comment]
Indeed, I managed to install the package manually! Thanks for the excellent support @steve-s, feel free to close or rename the issue
